Sources/ComposableArchitecture/Documentation.docc/Tutorials/BuildingSyncUps/06-SyncUpDetail/SyncUpDetail.tutorial

```diff
@@ -37,7 +37,7 @@
       }
       
       Next we add a case to the `Action` enum for each action the user can perform in the view.
-      Currently there are 3 buttons, and so we will have an action for each.
+      Currently there are 2 buttons, and so we will have an action for each.
```

Comment:
From this, which today just covers delete and edit.
https://github.com/pointfreeco/swift-composable-architecture/blob/af38644542b1310c0bb023a5af2545beb8854009/Sources/ComposableArchitecture/Documentation.docc/Tutorials/BuildingSyncUps/06-SyncUpDetail/SyncUpDetail.tutorial#L43-L44